server/api/region.go

```diff
@@ -168,9 +192,9 @@ func InitRegion(r *core.RegionInfo, s *RegionInfo) *RegionInfo {
 	s.ApproximateSize = r.GetApproximateSize()
 	s.ApproximateKeys = r.GetApproximateKeys()
 	s.ReplicationStatus = fromPBReplicationStatus(r.GetReplicationStatus())
+	s.Buckets = nil
```

Comment:
Is it necessary?

Reply:
If Buckets is empty, maybe the last Buckets value is retained.